multicluster/controllers/multicluster/leader/resourceexport_controller.go

```diff
@@ -461,7 +462,10 @@ func (r *ResourceExportReconciler) updateResourceExportStatus(resExport *mcsv1al
 
 // deleteResourceExport removes ResourceExport finalizer string and updates it, so Kubernetes can complete deletion.
 func (r *ResourceExportReconciler) deleteResourceExport(resExport *mcsv1alpha1.ResourceExport) (ctrl.Result, error) {
-	resExport.SetFinalizers(common.RemoveStringFromSlice(resExport.Finalizers, constants.ResourceExportFinalizer))
```

Comment:
We probably can remove this 'RemoveStringFromSlice' function after removing the caller in the stale_controller.go. Could you double check? Thanks.

Reply:
I was still used somewhere else and I changed that usage to `slices.DeleteFunc` as well. Removed the util function as suggested